install.sh

```diff
@@ -47,8 +47,17 @@ echo "Deno was installed successfully to $exe"
 if command -v deno >/dev/null; then
 	echo "Run 'deno --help' to get started"
 else
-	echo "Manually add the directory to your \$HOME/.bash_profile (or similar)"
+	case $SHELL in
+	/bin/zsh)
+		shell_profile_filename=".zshrc"
+		break
+		;;
+	*)
+		shell_profile_filename=".bash_profile"
+		;;
+	esac
```

Comment:
Could you format the case statements like the case statements above? e.g.

```bash
case $(uname -s) in
Darwin) os="osx" ;;
*) os="linux" ;;
esac
```

I think `shell_profile_filename` could be shortened to `shell_profile`.

Otherwise, LGTM.

Reply:
Hi,

The ci test runner forces `shfmt` (shell auto formatter) to make pretty.
https://github.com/denoland/deno_install/blob/master/.github/workflows/ci.yml

Reply:
Well, `shfmt` accepts both

```bash
case $SHELL in
/bin/zsh) shell_profile=".zshrc" ;;
*) shell_profile=".bash_profile" ;;
esac
```

```bash
case $SHELL in
/bin/zsh)
	shell_profile_filename=".zshrc"
	break
	;;
*)
	shell_profile_filename=".bash_profile"
	;;
esac
```
as "properly" formatted.

Again, I'd prefer the first formatting since it's similar to the formatting of the rest of the script.

Reply:
@MarkTiedemann 

I just made 2 new commits to update the variable to `shell_profile` and also adjusted the case statement to use the similar formatting as rest of the script. 

We should be good to merge:

🙏

Reply:
Yep, thanks, @jremi!